src/Apps/IdentityVerification/__tests__/routes.test.tsx

```diff
@@ -0,0 +1,43 @@
+import { IdentityVerificationPageQueryResponseFixture } from "Apps/IdentityVerification/__fixtures__/routes_IdentitifcationPageQuery"
+import { createMockNetworkLayer2 } from "DevTools"
+import { Resolver } from "found-relay"
+import {
+  FarceElementResult,
+  FarceRedirectResult,
+  getFarceResult,
+} from "found/lib/server"
+import { Environment, RecordSource, Store } from "relay-runtime"
+
+import { routes } from "Apps/IdentityVerification/routes"
+
+import { routes_IdentityVerificationPageQueryRawResponse } from "__generated__/routes_IdentityVerificationPageQuery.graphql"
+import { createRender } from "found"
+
+describe("IdentityVerification/routes", () => {
+  const idvID = (IdentityVerificationPageQueryResponseFixture.me as any)
```

Comment:
are we sure the cast to `any` is needed here? Based on my read of the code, the types check out.